Source/CMakeLists.txt

```diff
@@ -48,7 +48,17 @@ set(SRCS ${DolphinProcessSrc}
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
-find_package(Qt6Widgets REQUIRED)
+if (WIN32)
+  find_package(Qt6Widgets QUIET)
```

Comment:
Nit: I think a 4-space indentation is generally used in CMake files (and also in this file, except for a few poorly indented lines).